vision_layer/vision_tasks/src/markerDropper.cpp

```diff
@@ -0,0 +1,164 @@
+#include "ros/ros.h"
+#include "sensor_msgs/Image.h"
+#include "opencv2/imgproc/imgproc.hpp"
+#include "opencv2/core/core.hpp"
+#include "opencv2/imgproc/imgproc_c.h"
+#include "opencv2/highgui/highgui.hpp"
+#include <cv_bridge/cv_bridge.h>
+#include <image_transport/image_transport.h>
+#include <dynamic_reconfigure/server.h>
+#include <geometry_msgs/PointStamped.h>
+#include <sensor_msgs/image_encodings.h>
+#include <bits/stdc++.h>
+#include <stdlib.h>
+#include <string>
+
+#include <vision_tasks/markerDropperRangeConfig.h>
+#include <vision_commons/filter.h>
+#include <vision_commons/contour.h>
+#include <vision_commons/morph.h>
+#include <vision_commons/threshold.h>
```

Comment:
No need to include these headers if you have already included them in `markerDropper.h`.